server/src/main/java/org/apache/druid/server/security/AuthorizationUtils.java

```diff
@@ -359,6 +359,51 @@ public static Access authorizeAllResourceActions(
     return filteredResources;
   }
 
+  public static List<ResourceAction> makeSuperUserPermissions()
+  {
+    ResourceAction datasourceR = new ResourceAction(
+        new Resource(".*", ResourceType.DATASOURCE),
```

Comment:
Given that this pattern, ".*", is used frequently, does it make sense to create a specific string named constant for it?

Reply:
I thought about that, but anything I could think to name it would be more characters than the pattern, and the pattern seems unlikely to change for this use case, so ended up leaving it like it was. I don't feel very strongly about it though so can change it too.